### PR TITLE
Support user-provided installer args

### PR DESCRIPTION
**Pull request by tristanlabelle**
This allows specifying components to install or not in CI scripts.

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -24,6 +24,12 @@ inputs:
     description: 'Optional Github token for fetching a release. Only specify when using custom toolchains from non-public Github releases.'
     required: false
 
+  # common
+  installer-args:
```
Comment (tristanlabelle) [inline action.yml:28]: @compnerd Do we care to support this on macOS? It looks like there's an installer-like executable there too

**Comment by tristanlabelle** [comment]
@compnerd Would you be so nice as to run the tests for me, as I cannot?

File: action.yml (hunk 1 above)
Comment (compnerd) [reply action.yml:28]: Yes, we should support installation on macOS as well. I don’t think that the installer has arguments there though.

Comment (tristanlabelle) [reply action.yml:28]: That's what I meant, then I don't need more code in this PR.

**Pull request merged**